shopfloor/services/zone_picking.py

```diff
@@ -465,11 +475,15 @@ def list_move_lines(self):
         """
         return self._list_move_lines(self.zone_location)
 
-    def _list_move_lines(self, location, product=False):
+    def _list_move_lines(self, location, product=False, sublocation=False):
         move_lines = self._find_location_move_lines(location, product=product)
-        return self._response_for_select_line(move_lines, product=product)
+        return self._response_for_select_line(
+            move_lines, product=product, sublocation=sublocation
```

Comment:
```suggestion
            move_lines, product=product, sublocation=location
```

Reply:
No because this is to pass the sublocation to the frontend

Reply:
This is to pass the sublocation to the front end